libs/watt/src/lib/utils/date/watt-date.pipe.spec.ts

```diff
@@ -44,6 +44,11 @@ describe(WattDatePipe, () => {
     expect(pipe.transform(range)).toBe('25-03-2019 ― 27-03-2019');
   });
 
+  it('transforms date range with end null', () => {
+    const range = { start: '2019-03-25T22:00:00Z', end: '9999-12-31T23:59:59Z' };
+    expect(pipe.transform(range)).toBe('25-03-2019');
+  });
+
```

Comment:
suggestion: noticed that we're missing a test for the same day

```suggestion
  it('transforms date range with end null', () => {
    const range = { start: '2019-03-25T22:00:00Z', end: '9999-12-31T23:59:59Z' };
    expect(pipe.transform(range)).toBe('25-03-2019');
  });

  it('transforms date range with start and end dates falling on the same day', () => {
    const range = { start: '2019-03-25T22:00:00Z', end: '2019-03-25T22:59:59Z' };
    expect(pipe.transform(range)).toBe('25-03-2019');
  });
```

Reply:
Good catch